.editorconfig

```diff
@@ -0,0 +1,198 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+indent_style = space
+tab_width = 4
+
+# New line preferences
+end_of_line = crlf
```

Comment:
Can we change this to `lf`? The majority of Yarn Spinner developers work on macOS.

Reply:
dd32b0ee752f2df4f6e26b9a76002c19fcbaa51b